src/js/component/panel/PanelIssues.js

```diff
@@ -50,72 +68,228 @@ const defaultProps = {
-
-            if (isOwnedBySomeoneElse && props.hideIfOwnedBySomeoneElse) {
+            if (isOwnedBySomeoneElse && hideIfOwnedBySomeoneElse) {
                 return false;
             }
-
             return true;
         });
     }
 
-    // We need to be sure to filter the data if the user has set any filters
-    if (props.applyFilters && props.filters && !_.isEmpty(props.filters)) {
-        filteredData = _.filter(props.data, (item) => {
+    // Apply filters
+    if (applyFilters && filters && !_.isEmpty(filters)) {
+        preparedIssues = _.filter(preparedIssues, (item) => {
             const isImprovement = _.findWhere(item.labels, {name: 'Improvement'});
             const isTask = _.findWhere(item.labels, {name: 'Task'});
             const isFeature = _.findWhere(item.labels, {name: 'NewFeature'});
-            const isOnMilestone = item.milestone && item.milestone.id === props.filters.milestone;
+            const isOnMilestone = item.milestone && item.milestone.id === filters.milestone;
 
             // If we are filtering on milestone, remove everything not on that milestone
-            if (props.filters.milestone && !isOnMilestone) {
+            if (filters.milestone && !isOnMilestone) {
                 return false;
             }
-
-            return (props.filters.improvement && isImprovement)
-                || (props.filters.task && isTask)
-                || (props.filters.feature && isFeature);
+            return (filters.improvement && isImprovement)
+                || (filters.task && isTask)
+                || (filters.feature && isFeature);
         });
     }
 
-    if (!_.size(props.data) && props.hideOnEmpty) {
-        return null;
+    // Sort by priority, then owner
+    preparedIssues = _.sortBy(preparedIssues, (item) => {
+        const priority = priorities[item.url ?? ''] && (priorities[item.url].priority !== undefined)
+            ? priorities[item.url].priority
+            : Number.MAX_SAFE_INTEGER;
+        return [priority, item.currentUserIsOwner ? 0 : 1];
+    });
+
+    // Use localOrder if available, while waiting for Onyx to update
+    if (localOrder.length && localOrder.length === preparedIssues.length) {
+        const dataById = _.indexBy(preparedIssues, 'id');
+        return _.filter(_.map(localOrder, id => dataById[id]), Boolean);
     }
+    return preparedIssues;
+}
+
+function SortableIssue(props) {
+    const {issue} = props;
+    const {
+        attributes,
+        listeners,
+        setNodeRef,
+        transform,
+        transition,
+        isDragging,
+    } = useSortable({id: issue.id.toString()});
+
+    const style = {
+        transform: CSS.Transform.toString(transform),
+        transition,
+        zIndex: isDragging ? 2 : 1,
+        background: isDragging ? '#f0f0f0' : undefined,
+    };
+
+    return (
+        <div
+            ref={setNodeRef}
+            style={style}
+            {...attributes} // eslint-disable-line react/jsx-props-no-spreading
```

Comment:
Please put the comments on the line above, and also add comments to explain why the rules are being disabled.